examples/pipeline/kg_builder_from_text.py

```diff
@@ -76,10 +75,8 @@ async def main(neo4j_driver: neo4j.Driver) -> PipelineResult:
     pipe = Pipeline()
     # define the components
     pipe.add_component(
-        LangChainTextSplitterAdapter(
-            # chunk_size=50 for the sake of this demo
-            CharacterTextSplitter(chunk_size=50, chunk_overlap=10, separator=".")
-        ),
+        # chunk_size=50 for the sake of this demo
+        FixedSizeSplitter(chunk_size=50, chunk_overlap=10),
```

Comment:
How does the performance compare to LangChainTextSplitterAdapter anecdotally?

Reply:
In terms of speed or in terms of how it chunks? In terms of speed I haven't noticed any differences although I haven't tested it with any very large documents. I doubt though that it'd be significantly slower than the LangChain splitters. In terms of how it splits it's nice in that it always gives you chunks less than the `chunk_size` as opposed to LangChain splitters such as the `CharacterTextSplitter` which have a habit of always creating a few chunks that are greater than the chunk size (they won't raise an error and will just log `Created a chunk of size 101, which is longer than the specified 100` when they do this). I wouldn't recommend this chunker for much beyond testing and simple demos though. For production use cases something like the `RecursiveCharacterTextSplitter` should probably be used.